src/styles/normalize.scss

```diff
@@ -0,0 +1,6 @@
+* {
```

Comment:
this is `reset`)

Reply:
https://elad.medium.com/normalize-css-or-css-reset-9d75175c5d1e

Reply:
thx)